test/integration/share/README.md

```diff
@@ -0,0 +1,20 @@
+# InSpec Profile: `share`
```

Comment:
This file differs substantially with `test/integration/share/README.md` in other formulas (which are already managed by the `ssf-formula`).  However, the changes here are very useful, so we'd need to update the template to include these.  One formula where this would get a little more confusing is the [libvirt-formula](https://github.com/saltstack-formulas/libvirt-formula/blob/6f1400e0021f16765f774aaecf58eda03fbebb4a/test/integration/share/README.md), where there are multiple libraries.  It would be extremely helpful if we could standardise these so that the template can be updated accordingly.